cves/kernel/CVE-2020-11494.yml

```diff
@@ -84,18 +86,12 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
-  note:
-- commit:
-  note:
 - commit: '08fadc32ce6239dc75fd5e869590e29bc62bbc28'
   note: |
-    Taken from NVD references list with Git commit. If you are
-    curating, replace this comment with 'Manually confirmed'
+    Manually confirmed. This commit is not reachable from any branch
```

Comment:
This commit is not reachable so would this not be the commit that fixes it?

Reply:
It fixed the issue but it might be a local fix or a patch that hasn't been merged into the main branch.